src/hitchhiker/konserve.cljc

```diff
@@ -0,0 +1,162 @@
+(ns hitchhiker.konserve
+  (:refer-clojure :exclude [resolve subvec])
+  (:require [clojure.core.rrb-vector :refer [catvec subvec]]
+            #?(:clj [clojure.core.async :refer [chan promise-chan put!] :as async]
+               :cljs [cljs.core.async :refer [chan promise-chan put!] :as async])
+            [konserve.core :as k]
+            [konserve.memory :refer [new-mem-store]]
+            [hasch.core :refer [uuid]]
+            [clojure.set :as set]
+            #?(:clj [hitchhiker.tree.core :refer [go-try <?] :as core]
+               :cljs [hitchhiker.tree.core :as core])
+            [hitchhiker.tree.messaging :as msg])
+  #?(:cljs (:require-macros [hitchhiker.tree.core :refer [go-try <?]])))
+
```

Comment:
Could you add either an `.md` to `docs/` or a comment here that explains how data is addressed and represented? Some things to describe would be what valid keys are and what read handlers are.